tests/integration/test_analysisd/test_signature_id/data/test_cases/cases_null_signature_id.yaml

```diff
@@ -0,0 +1,5 @@
+- name: Null Signature ID - wazuh restarts
```

Comment:
```suggestion
- name: Null Signature ID
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3649/commits/cf8e8008bb6aaf5eaf95b75a4c64f3757fba2713